### PR TITLE
Create input form 

### PR DESCRIPTION
**Pull request by Aya74**
-  Edit page2.html by adding a form in it that contain:
   -  Text input to add the tittle of the task.
   -   Date input to add the date that the task should be done in.
   -   Time input to add the time that the task should be done in.
   -   Text input to add the description of the task.
   -   Text input to add additional notes.
   -   Cancel link to retract of adding task and its shaped to look like button using css.
   -   Button type submit named done to save the task.

- Edit page2.css by by adding:
  -  Font size (Amatic SC).
  - Background color (rich black).
  -  Color to the text and the buttons (white) .
  - Background color for buttons (Cancel button with color red an Done button with color blue).
  -  Edit the position of the elements.

- The changes have been working out (the path of the file, the outline of the inputs, the cancel button).

relates #7

### CODE REVIEW

**Review by Yousef-Ahmad1997128 — APPROVED** [review]

**Review by ferassmohana — APPROVED** [review]

File: html/page2.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="/css/page2.css" />
```
Comment (sara219) [inline html/page2.html]: the path is wrong, its not working 

```html
    <link rel="stylesheet" href="../css/page2.css" />
```

File: css/page2.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+/* Add style to the form */
+.form-container {
+  padding: 10px;
+}
+
+/* Add style to the input in the form-container form */
+.form-container input[type="text"],
+.form-container input[type="date"],
+.form-container input[type="time"] {
```
Comment (sara219) [inline css/page2.css:20]: for every input, remove the outline 

Screenshot from 2021-07-16 19-27-39

**Review by sara219 — CHANGES_REQUESTED** [review]

File: html/page2.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+            type="text"
+            name="notes"
+            id="notesInput"
+            placeholder="Enter additional notes"
+          />
+        </div>
+
+        <div class="button">
+          <a href="" id="cancel">Cancel</a>
+          <button type="submit" id="submitButton">Done</button>
```
Comment (sara219) [inline html/page2.html]: why you add the cancel button as a hyperlink instead of a button

**Review by sara219 — CHANGES_REQUESTED** [review]

File: html/page2.html (hunk 3 above)
Comment (Aya74) [reply html/page2.html:58]: I didn't found a button type attribute (cancel) and when I research about it I found that it's recommend to make it as hyperlink and style it to look like button.

Comment (Aya74) [reply html/page2.html]: I didn't found a button type attribute (cancel) and when I research about it I found that it's recommended to make it use hyperlink and style it to look like a button.
Should I make it like ````<button><a href="main.html">cancel</a></button>```` ?
Or what's your recommendation?

Comment (sara219) [reply html/page2.html]: you can add it like  

```html
<input type="button" name="cancel" value="cancel" onClick="window.location.href='main.html" />
```

Comment (Aya74) [reply html/page2.html]: Ok, thanks

**Review by MahmoudJD95 — APPROVED** [review]

**Review by sara219 — APPROVED** [review]

**Pull request merged**